src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp

```diff
@@ -60,7 +60,9 @@ enum EltwiseTypes {
     BITWISE_AND,
     BITWISE_NOT,
     BITWISE_OR,
-    BITWISE_XOR
+    BITWISE_XOR,
+    ELTWISE_MINIMUM,
```

Comment:
Min and Max are part of MinMaxOpType type and are used in MaxMinLayerTest. We either need to use MinMaxOpType enum to test ARM JIT functionality or fully unite it with EltwiseTypes (remove MaxMinLayerTest test and use Eltwise test instead)

Reply:
discussed offline, was removed